Integrations/Cymulate/Cymulate.py

```diff
@@ -0,0 +1,400 @@
+            'Timestamp : '', 'InProgress: ''}...] }
+        """
+
+        get_ids_route = ''
+
+        if module_type == CymulateModuleTypeEnum.APT:
+            get_ids_route = 'apt/ids'
+        elif module_type == CymulateModuleTypeEnum.IMMEDIATE_THREATS:
+            get_ids_route = 'immediate-threats/ids'
+        elif module_type == CymulateModuleTypeEnum.HOPPER:
+            get_ids_route = 'hopper/ids'
+        elif module_type == CymulateModuleTypeEnum.PHISHING:
+            get_ids_route = 'phishing/ids'
+        elif module_type == CymulateModuleTypeEnum.WAF:
+            get_ids_route = 'waf/ids'
+        elif module_type == CymulateModuleTypeEnum.DLP:
+            get_ids_route = 'dlp/ids'
+        elif module_type == CymulateModuleTypeEnum.BROWSING:
+            get_ids_route = 'browsing/ids'
+        elif module_type == CymulateModuleTypeEnum.EDR:
+            get_ids_route = 'edr/ids'
+        elif module_type == CymulateModuleTypeEnum.MITRE:
+            get_ids_route = 'mitre/ids'
+        elif module_type == CymulateModuleTypeEnum.MAIL:
+            get_ids_route = 'mail/ids'
+
+        results = self._http_request(
+            method='GET',
+            url_suffix=get_ids_route,
+            params={'from': from_date}
+        )
+
+        return results['data']
+
+    def get_attack_technical_info(self, module_type, incident_id):
+        """ Get technical Data about incident.
+        Args:
+            module_type (CymulateModuleTypeEnum): module_type enum
+            incident_id (string): The incident id
+
+        Returns:
+            JSON - https results struct of { 'success' : true/false, 'data' : [{'Id' : '', 'Name' : '',
+            'Timestamp : '', 'InProgress: ''}...] }
+        """
+
+        technical_route = ''
+
+        if module_type == CymulateModuleTypeEnum.APT:
+            technical_route = 'apt/attack/technical'
+        elif module_type == CymulateModuleTypeEnum.IMMEDIATE_THREATS:
+            technical_route = 'immediate-threats/attack/technical'
+        elif module_type == CymulateModuleTypeEnum.HOPPER:
+            technical_route = 'hopper/attack/technical'
+        elif module_type == CymulateModuleTypeEnum.PHISHING:
+            technical_route = 'phishing/attack/technical'
+        elif module_type == CymulateModuleTypeEnum.WAF:
+            technical_route = 'waf/attack/technical'
+        elif module_type == CymulateModuleTypeEnum.DLP:
+            technical_route = 'dlp/attack/technical'
+        elif module_type == CymulateModuleTypeEnum.BROWSING:
+            technical_route = 'browsing/attack/technical'
+        elif module_type == CymulateModuleTypeEnum.EDR:
+            technical_route = 'edr/attack/technical'
+        elif module_type == CymulateModuleTypeEnum.MITRE:
+            technical_route = 'mitre/attack/technical'
+        elif module_type == CymulateModuleTypeEnum.MAIL:
+            technical_route = 'mail/attack/technical'
```

Comment:
all these seems to be different routes, are they return different response structure?

Reply:
now we support only 'immediate-threats'